torobo/torobo.xml

```diff
@@ -0,0 +1,1042 @@
+<mujoco model="torobo">
+  <compiler angle="radian" meshdir="assets" autolimits="true"/>
+
+  <option timestep="0.001" integrator="implicitfast"/>
+
+  <default>
+    <default class="torobo">
+      <joint armature="0.1" stiffness="0" damping="0"/>
```

Comment:
Can you remove attributes that are defaults already in MuJoCo? e.g. stiffness and damping are 0 so feel free to remove.